drivers/charger/Kconfig

```diff
@@ -0,0 +1,24 @@
+# Charger configuration options
+
```

Comment:
Kconfigs needs to be used and should come with the driver/subsystem they support.